### PR TITLE
Added 10 CVEs

### PR DESCRIPTION
**Pull request by DellEMCProductSecurity**

### CODE REVIEW

File: 2020/29xxx/CVE-2020-29505.json [hunk 1]
```diff
@@ -1,17 +1,70 @@
 {
```
Comment (cve-team) [inline 2020/29xxx/CVE-2020-29505.json:1]: None is not of type 'string'

File: 2020/29xxx/CVE-2020-29506.json [hunk 2]
```diff
@@ -1,17 +1,70 @@
 {
```
Comment (cve-team) [inline 2020/29xxx/CVE-2020-29506.json:1]: None is not of type 'string'

File: 2020/29xxx/CVE-2020-29507.json [hunk 3]
```diff
@@ -1,17 +1,70 @@
 {
```
Comment (cve-team) [inline 2020/29xxx/CVE-2020-29507.json:1]: None is not of type 'string'

File: 2020/29xxx/CVE-2020-29508.json [hunk 4]
```diff
@@ -1,17 +1,70 @@
 {
```
Comment (cve-team) [inline 2020/29xxx/CVE-2020-29508.json:1]: None is not of type 'string'

File: 2020/35xxx/CVE-2020-35169.json [hunk 5]
```diff
@@ -1,17 +1,70 @@
 {
```
Comment (cve-team) [inline 2020/35xxx/CVE-2020-35169.json:1]: None is not of type 'string'

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by DellEMCProductSecurity** [comment]
Please review again.

**Pull request merged**